src/index.html

```diff
@@ -3,83 +3,49 @@
   <head>
     <meta charset="utf-8">
     <title>Webpack Starter Kit</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Poppins&display=swap" rel="stylesheet">
   </head>
   <body>
-    <header>
-      <h1>Travel Tracker</h1>
-      <h2 class="welome-banner"></h2>
+    <header class="sticky">
+    <span class="logo">
+      <img class="travel" src="/images/travel.svg" alt="plane and globe" aria="hidden">
+      <h1 class="page-title">Travel Tracker</h1>
+    </span>
+       <button class="display-booking-button hidden">
+       <img class="airplane" src="/images/airplane.svg" alt="airplane" aria="hidden"><br>Book a trip!</button>
     </header> 
-
+  
     <main>
       <section class="login">
-
-        <form class="login-form">
-          <fieldset>
+        <form class="login-form" for="login-button" onsubmit="">
+          <fieldset class="login-field">
             <legend>Login</legend>
-            <label for="username">Username<strong><abbr title="required">*</abbr></strong>
+            <label for="username">Username <strong><abbr title="required">*</abbr></strong>
```

Comment:
Good case use for strong! A screen reader will overlay emphasis on this now as opposed to using css to style it.